sapi/litespeed/lsapi_main.c

```diff
@@ -448,7 +448,7 @@ static int sapi_lsapi_activate()
 static sapi_module_struct lsapi_sapi_module =
 {
     "litespeed",
-    "LiteSpeed V7.3.2",
+    "LiteSpeed" PHP_VERSION,
```

Comment:
I don't think it makes sense to include the PHP version here, and I don't think we do this for other SAPIs either. Just dropping it should be fine.

Also, as written you'll get something like `LiteSpeed7.4.0` due to missing space.